README.md

```diff
@@ -20,17 +16,29 @@ $ cd <folder>
 ```
 
 ```
-$ git clone https://github.com/garciajordy/Enumerable-Methods.git
+$ git clone https://github.com/JulianaOsemeke/tic_tac_toe
 ```
 
-Testing
+## Instructions
 
-To test the code, run `rspec` from root of the folder using terminal.
+```
+1)The game is played on a grid that's 3 squares by 3 squares.
+```
 
-> Rspec is used for the test.
+```
+2)Write a number between 1 and 9
 
-```bash
-$ gem install rspec
+to select the row and column respectively that you want to play on.
+```
+
+```
+3)Once a player gets 3 marks in a row
+
+(up, down, across, or diagonally) is the winner.
+```
+
+```
+4)If all 9 squares are full the game is over.
```

Comment:
Nice to have a Readme file at this point, unfortunately, it didn't help me to play the game, kindly help the one who clones your repo to understand all the steps he needs to follow to make it work

Reply:
We add more specific instructions like requested